awkward/array/jagged.py

```diff
@@ -408,9 +409,17 @@ def parents(self, value):
 
     @property
     def index(self):
```

Comment:
As I've said earlier, I've got to rename this function so that it doesn't conflict with `index` on other array types.

Reply:
`localindex` ?

Reply:
Just a comment; nothing to do here.

Reply:
That would be a good name. It was originally `index` because it was motivated for making Pandas indexes. Apart from that context, it is local and not global.